### PR TITLE
MRC-1792: Add model calibration options

### PR DESCRIPTION
**Pull request by r-ash**
This PR will
* Add a function for returning model calibration options

Note that this PR dupes calibration options in model run options and model calibration options. The PR to remove options from the model run section I will add separately as removing those is blocked on updating the front end

### CODE REVIEW

File: inst/metadata/calibration_run_options.json [hunk 1]
```diff
@@ -0,0 +1,165 @@
+{
```
Comment (richfitz) [inline inst/metadata/calibration_run_options.json:1]: I feel we use spaces elsewhere for indentation - might be worth updating this so we don't end up with a weird mix of tabs and spaces

**Review by richfitz — APPROVED** [review]
This looks fine to me, so long as @jeffeaton is happy with the result. I'd probably un-tabbify that json file though

File: inst/metadata/calibration_run_options.json (hunk 1 above)
Comment (r-ash) [reply inst/metadata/calibration_run_options.json:1]: Thanks, hadn't noticed that but I've switched all tabs in options to 2 spaces.

**Comment by r-ash** [comment]
Going to merge this as is - the options can be updated and finalised whenever is convenient

File: inst/metadata/calibration_run_options.json (hunk 1 above)
Comment (jeffeaton) [reply inst/metadata/calibration_run_options.json:1]: I think the main culprit here is my Emacs config defaulting to tabs and sneaking them in there!! I need to figure that out.

**Review by jeffeaton — COMMENTED** [review]
This looks great to me -- thanks very much.

**Pull request merged**